specification/hardwaresecuritymodules/resource-manager/Microsoft.HardwareSecurityModules/preview/2018-10-31-preview/dedicatedhsm.json

```diff
@@ -392,6 +417,45 @@
     }
   },
   "definitions": {
+    "DedicatedHsmOperation": {
+      "description": "REST API operation",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The name of the Dedicated HSM Resource Provider Operation.",
+          "type": "string"
+        },
+        "display": {
+          "description": "The object that represents the operation.",
+          "properties": {
+            "provider": {
+              "description": "The Resource Provider of the operation",
+              "type": "string"
+            },
+            "resource": {
+              "description": "Resource on which the operation is performed.",
+              "type": "string"
+            },
+            "operation": {
+              "description": "Operation type: Read, write, delete, etc.",
+              "type": "string"
+            }
```

Comment:
This is missing the `description` property as defined in the contract [here](https://github.com/Azure/azure-resource-manager-rpc/blob/fbb33d93f0b2cc58ef345c3a430766632ad23936/v1.0/proxy-api-reference.md#exposing-available-operations). Please could you add this?

Reply:
Hi @anthony-c-martin , the new changes are here: